cmd/builder/internal/config/default.yaml

```diff
@@ -17,4 +17,6 @@ extensions:
 processors:
   - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.68.0
   - gomod: go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.68.0
-
+connectors:
+  - import: go.opentelemetry.io/collector/connector/forwardconnector
+    gomod: go.opentelemetry.io/collector/connector/forwardconnector v0.68.0
```

Comment:
Is the `import` entry needed here?  If the package import path and module identifier are equal I think you only need `gomod`.

Reply:
Good call. Fixed.